byon/files/seaclouds-catalog.bom

```diff
@@ -333,8 +329,6 @@ brooklyn.catalog:
             deployer.port: $brooklyn:component("deployer").attributeWhenReady("brooklynnode.webconsole.httpPort")
             sla.host: $brooklyn:component("sla-core").attributeWhenReady("host.address")
             sla.port: $brooklyn:component("sla-core").attributeWhenReady("http.port")
-            monitor.manager.host: $brooklyn:component("monitor-manager").attributeWhenReady("host.address")
-            monitor.manager.port: $brooklyn:component("monitor-manager").config("monitor.manager.port")
```

Comment:
Dashboard does not already require Monitor-manager because rules will be deployed by the Deployer. If you take a look to SeaCloudsManagementPolicy you will see that monitor information is added as policy's configuration.